cogs/file_cog.py

```diff
@@ -46,26 +48,27 @@ class FileCog(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
-    def __init__(self, bot):
-        self.bot = bot
-
     @commands.Cog.listener()
-    async def on_message(self, message):
+    async def on_message(self, message: discord.Message):
```

Comment:
Add void return type:
```suggestion
    async def on_message(self, message: discord.Message) -> None:
```